common/src/main/java/com/wynntils/utils/wynn/ColorScaleUtils.java

```diff
@@ -18,29 +20,25 @@
 import net.minecraft.network.chat.TextColor;
 
 public final class ColorScaleUtils {
+    private static final ItemStatInfoFeature isif = Managers.Feature.getFeatureInstance(ItemStatInfoFeature.class);
```

Comment:
I don't think I have to mention how this won't fly at all. You should cache the maps in the feature, or figure out an alternative solution.